src/utils/Client.util.ts

```diff
@@ -63,6 +67,56 @@ export function createHTTPMetadataQueryParam(metadata: KeyValueType = {}): strin
   return queryParam;
 }
 
+/**
+ * Converts a set of optional parameters into a query string
```

Comment:
@shubham1172 maybe we should have a more generic util for query param building? wdyt?